components/portal/messages/controllers.js

```diff
@@ -205,13 +205,19 @@ define(['angular'], function(angular) {
             // Else hide messages features
             if (angular.equals($scope.$parent.showMessagesFeatures, true)) {
               configureNotificationsScope();
+              checkForChangesInOtherInstances;
```

Comment:
:question: does this need parenthesis to be called?

Reply:
The committer yelled at me when I didn't assign the $rootScope.$watch to a variable, then it yelled at me when I assigned it to a variable, but didn't reference it.

Referencing it in line 208 made my problems go away. 

var checkForChangesInOtherInstances = $rootScope.$watch(
          'priorityNotificationCount', function() {
            configurePriorityNotificationsScope();
        });

Reply:
Thats coming from https://github.com/Gillespie59/eslint-plugin-angular/blob/master/docs/rules/on-watch.md

the variable is to remove the watcher registration.
if the watcher is never unregistered that's a potential red flag.